src/dlgAboutDialog.cpp

```diff
@@ -1062,3 +1070,18 @@ void dlgAboutDialog::setSupportersTab(const QString& htmlHead)
     textBrowser_supporters->setDocument(supportersDocument.get());
     textBrowser_supporters->setOpenExternalLinks(true);
 }
+
+QString dlgAboutDialog::createBuildInfo() const {
+
+    QString buildInfo = tr("<table border=\"0\" style=\"margin-bottom:36px; margin-left:36px; margin-right:36px;\" width=\"100%\" cellspacing=\"2\" cellpadding=\"0\">\n"
```

Comment:
These strings may need further refinements as well.    Now or later, we need to make sure the strings for translation (those wrapped with `tr()` ) contain as little markup as possible in order to ease the translation process.    
I believe the markup/formatting strings should be made into `QStringLiteral`s as well, following other conventions we use.

Reply:
Sure, will correct that for the new code.